src/managing-workflow/upgrading-workflow.md

```diff
@@ -67,27 +67,27 @@ $ kubectl --namespace=deis get secret database-creds -o yaml > ~/active-deis-dat
 $ kubectl --namespace=deis get secret builder-ssh-private-keys -o yaml > ~/active-deis-builder-secret-ssh-private-keys.yaml
 
 # fetch new chart
-$ helmc fetch deis/workflow-rc1
+$ helmc fetch deis/workflow-rc2
 
 # update your off-cluster storage configuration
 $ $EDITOR $(helmc home)/workspace/charts/workflow-beta4/tpl/generate_params.toml
 
 # generate new templates
-$ helmc generate -x manifests workflow-rc1
+$ helmc generate -x manifests workflow-rc2
 
 # copy your active database secrets into the helmc workspace
 $ cp ~/active-deis-database-secret-creds.yaml \
-	$(helmc home)/workspace/charts/workflow-rc1-manifests/deis-database-secret-creds.yaml
+	$(helmc home)/workspace/charts/workflow-rc2-manifests/deis-database-secret-creds.yaml
 
 # copy your active builder ssh keys into the helmc workspace
 $ cp ~/active-deis-builder-secret-ssh-private-keys.yaml \
-	$(helmc home)/workspace/charts/workflow-rc1/manifests/deis-builder-secret-ssh-private-keys.yaml
+	$(helmc home)/workspace/charts/workflow-rc2/manifests/deis-builder-secret-ssh-private-keys.yaml
 
 # uninstall workflow
 $ helmc uninstall workflow-beta4 -n deis
```

Comment:
Ya... there are some beta4 refs that should have been updated. Finding and fixing.